webapp/src/routes/Home/ShowLifebanks.js

```diff
@@ -0,0 +1,316 @@
+    letterSpacing: "1",
+    textAlign: "center",
+    color: "#121212",
+  }
+}))
+
+const ShowLifebanks = ({ banks, loading, isDesktop }) => {
+  const classes = useStyles()
+
+  const LoadBanks = () => {
+    return (
+      <>
+        {loading &&
+          <Box className={classes.wrapper}>
+            <CircularProgress />
+          </Box>
+        }
+        {!loading && banks.length <= 0 && (
+          <ListItem className={classes.listItem} >
+            <ListItemText
+              primary={
+                <Typography className={classes.listItemPrimaryText} noWrap variant="body2">No blood bank available</Typography>
+              }
+            />
+          </ListItem>
+        )}
+        {!loading && banks.length > 0 && banks.map(bank => (
+          <BankItem
+            key={bank.id}
+            id={bank.id}
+            name={bank.name}
+            description={bank.info.description}
+          />
+        ))}
+      </>
+    )
+  }
+
+  const BankItem = (props) => {
+    return (
+      <ListItem className={classes.listItem} button>
+        <ListItemAvatar>
+          <Avatar>
+            <LocalHospitalIcon />
+          </Avatar>
+        </ListItemAvatar>
+        <ListItemText
+          primary={
+            <Typography className={classes.listItemPrimaryText} noWrap variant="body2">{props.name}</Typography>
+          }
+          secondary={
+            <Typography className={classes.listItemSecondaryText} noWrap variant="body2">{props.description}</Typography>
+          }
+        />
+      </ListItem>
+
+    )
+  }
+
+  BankItem.propTypes = {
+    name: PropTypes.string,
+    description: PropTypes.string,
+  }
+
+  const LoadBanksDesktop = () => {
+    return (
+      <>
+        {loading &&
+          <Box className={classes.wrapper}>
+            <CircularProgress />
+          </Box>
+        }
+        {!loading && banks.length <= 0 && (
+          <Grid item xs={2}>
+            <Card className={classes.cardRoot}>
+              <Grid
+                container
+                direction="row"
+                justify="center"
+                alignItems="center"
+                style={{ height: "100%" }}
+              >
+                <Typography className={classes.title} color="textSecondary" gutterBottom>
+                  No blood bank available
+              </Typography>
+              </Grid>
+            </Card>
+          </Grid>
+        )}
+        {!loading && banks.length > 0 && banks.map(bank => (
+          <Grid item md={3} lg={2} key={bank.id}>
+            <BankCard
+              key={bank.id}
+              id={bank.id}
+              name={bank.name}
+              description={bank.info.description}
+            />
+          </Grid>
+        ))}
+      </>
+    )
+  }
+
+  const truncateString = (str) => {
+    const num = 55
+
+    if (str.length <= num) {
+      return str
+    }
+
+    return str.slice(0, num) + '...'
+  }
+
+
+  const BankCard = (props) => {
```

Comment:
When the components only has a return in its JSX content, you can do something like

```
const OfferItem = (props) => (
    JSX HERE
)
```